aws-lc-rs/src/data/agreement_tests.txt

```diff
@@ -250,31 +250,6 @@ Curve = P-521
 PeerQ = 0400D0B3975AC4B799F5BEA16D5E13E9AF971D5E9B984C9F39728B5E5739735A219B97C356436ADC6E95BB0352F6BE64A6C2912D4EF2D0433CED2B6171640012D9460F
 Error = Peer public key is missing the Y coordinate completely.
 
-Curve = P-256
```

Comment:
Instead of deleting these "negative" tests for public keys using the compressed key format, we should transform them into "positive" KATs.